bootstrapvz/common/tasks/grub.py

```diff
@@ -61,10 +99,14 @@ def run(cls, info):
 				device_map.write('(hd0) {device_path}\n'.format(device_path=device_path))
 				if not isinstance(p_map, partitionmaps.none.NoPartitions):
 					for idx, partition in enumerate(info.volume.partition_map.partitions):
-						device_map.write('(hd0,{prefix}{idx}) {device_path}\n'
-						                 .format(device_path=partition.device_path,
-						                         prefix=partition_prefix,
-						                         idx=idx + 1))
+						device_map.write(
+							'(hd0,{prefix}{idx}) {device_path}\n'
+							.format(
+								device_path=partition.device_path,
+								prefix=partition_prefix,
+								idx=idx + 1
+							)
+						)
```

Comment:
Ok, so I noticed the tox.ini file. Is this still being run by the TravisCI build? I only say it mention flake8.